scss/_patterns_strip.scss

```diff
@@ -114,3 +116,88 @@
     @extend %section-padding--deep;
   }
 }
+
+$color-suru-start: lighten($color-brand, 10%) !default;
+$color-suru-middle: $color-brand !default;
+$color-suru-end: darken($color-brand, 10%) !default;
+
+@mixin vf-p-strip-suru {
+  .p-strip--suru {
+    @extend %vf-strip;
+
+    background-blend-mode: multiply, multiply, normal, normal, normal;
+    background-image: linear-gradient(to bottom right, rgba(205, 205, 205, 0.55) 0%, rgba(205, 205, 205, 0.55) 49.8%, transparent 50%, transparent 100%),
+      linear-gradient(to bottom left, rgba(205, 205, 205, 0.55) 0%, rgba(205, 205, 205, 0.55) 49.8%, transparent 50%, transparent 100%),
+      linear-gradient(to top right, #fff 0%, #fff 49%, transparent 50%, transparent 100%), linear-gradient(#fff 0%, #fff 100%),
+      linear-gradient(111deg, $color-suru-start 10%, $color-suru-middle 37%, $color-suru-end 100%);
+    @supports not (background-blend-mode: multiply) {
```

Comment:
I think we should do it another way around, by default provide the styling for browsers not supporting blend mode multiply (because they may not support `@support` as well...) and use `@support` to detect browsers that do.

So something like:

```scss

background-image: // gradients without multiply

@supports (background-blend-mode: multiply) {
   background-blend-mode: multiply, multiply, normal, normal, normal;
   background-image: // gradients with multiply
}
```